spec/features/admin/orders/show_spec.rb

```diff
@@ -0,0 +1,124 @@
+require 'rails_helper'
```

Comment:
Apologies for the big ugly before(:each) blocks in this and other tests, but wanted to have sufficient data to fully test the ordering from story 32

Reply:
That's what before(:each) is for!

Reply:
Yeah, orders take up quite a bit